packages/tailwindcss/src/plugin-api.test.ts

```diff
@@ -1011,6 +1011,83 @@ describe('addUtilities()', () => {
       )
     }).rejects.toThrowError(/invalid utility selector/)
   })
+
+  test('supports multiple selector names', async () => {
+    let compiled = await compile(
+      css`
+        @plugin "my-plugin";
+        @tailwind utilities;
+
+        @theme reference {
+          --breakpoint-lg: 1024px;
+        }
+      `,
+      {
+        async loadPlugin() {
+          return ({ addUtilities }: PluginAPI) => {
+            addUtilities({
+              '.form-input, .form-textarea': {
+                appearance: 'none',
+                'background-color': '#fff',
+              },
+            })
+          }
+        },
+      },
+    )
+
+    expect(optimizeCss(compiled.build(['form-input', 'lg:form-textarea'])).trim())
+      .toMatchInlineSnapshot(`
+        ".form-input {
+          appearance: none;
+          background-color: #fff;
+        }
+
+        @media (width >= 1024px) {
+          .lg\\:form-textarea {
+            appearance: none;
+            background-color: #fff;
+          }
+        }"
+      `)
+  })
+
+  test('supports pseudo classes and pseudo elements', async () => {
+    let compiled = await compile(
+      css`
+        @plugin "my-plugin";
+        @tailwind utilities;
+
+        @theme reference {
+          --breakpoint-lg: 1024px;
+        }
+      `,
+      {
+        async loadPlugin() {
+          return ({ addUtilities }: PluginAPI) => {
+            addUtilities({
+              '.form-input, .form-textarea:hover:focus': {
```

Comment:
can we add `.form-input::placeholder` to this list too just to show that the pseudo-element works?